### PR TITLE
Add ocp manifest generation

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

**Comment by damdo** [comment]
Thanks for this PR!
Any reason not to have the manifests-generator installed in `/openshift/tools` similarly to what we do for the other providers and its manifests in `/openshift/manifests`?

We decided to have them there, rather than, for example, under `/hack/tools`, to avoid conflicts with upstream during rebases, which is a common problem when you have local changes in the same go.mod/sum/vendor files.

**Comment by honza** [comment]
Moved the relevant tools and manifests to `openshift/`.

**Comment by honza** [comment]
Good catch, thanks. Fixed.  Also removed an extra manifest file.

File: openshift/tools/bin/kustomize
Comment (dtantsur) [inline openshift/tools/bin/kustomize:1]: Do we really need the binary Kustomize here?

Comment (honza) [reply openshift/tools/bin/kustomize:1]: Fixed

**Pull request merged**